dotcom-rendering/src/web/components/RichLink.tsx

```diff
@@ -96,6 +95,11 @@ const richLinkTitle = css`
 	}
 `;
 
+const richLinkTitle = css`
+	${blogRichLinkTitle};
+	font-size: 14px;
+`;
```

Comment:
Semantically it's a bit confusing if `richLinkTitle` extends `blogRichLinkTitle` because in the switch statement I feel like I'm either going to get labs, blog or normal. But actually I end up with a combination of them.

We don't often use this pattern (where one style includes another one). Typically we have functions that return styles based on values passed in, e.g.:

```
const myStyle = (aValue: valueType) => css`
  color: ${decideColour(aValue)};
`
```

Would that work here?

Failing that I guess we could just duplicate things? A little duplication is no bad thing.